nodebuilder/tests/swamp/config.go

```diff
@@ -16,7 +16,7 @@ type Config struct {
 func DefaultConfig() *Config {
 	cfg := core.DefaultTestConfig()
 	// target height duration lower than this tend to be flakier
```

Comment:
```suggestion
	// timeout commit lower than this tend to be flakier
```

Reply:
[optional] we may not even need to override this value if the default is 300 milliseconds: https://github.com/rootulp/celestia-app/blob/0d93ec79bc65f8a2bb37c4bb8a1fb7140a033251/test/util/testnode/config.go#L139-L142

^ needs confirmation by running this test without lines 18 - 19